platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/Mapbox.java

```diff
@@ -126,7 +155,7 @@ public static synchronized Boolean isConnected() {
    */
   private static void initializeTelemetry() {
     try {
-      INSTANCE.telemetry = getModuleProvider().obtainTelemetry();
+      INSTANCE.telemetry = INSTANCE.moduleProvider.obtainTelemetry();
```

Comment:
This should still rely on the `getModuleProvider`, otherwise, we'll run into NPEs.